proxy/logging/LogAccess.cc

```diff
@@ -1264,6 +1264,26 @@ LogAccess::marshal_cache_lookup_url_canon(char *buf)
   return len;
 }
 
+/*-------------------------------------------------------------------------
+  -------------------------------------------------------------------------*/
+int
+LogAccess::marshal_client_sni_server_name(char *buf)
+{
+  // NOTE:  For this string_view, data() must always be nul-terminated, but the nul character must not be included in
+  // the length.
+  //
+  std::string_view sn = "n/a";
```

Comment:
returning `n/a` is non-standard for ATS. The default for no value ends up being `-` (which I think happens if you don't set anything in `buf`)